### DISCUSSION TITLE
it says its a virus so idk if i should download it

### DISCUSSION THREAD

**Discussion by yacmag** [nesrak1/UABEA#386] [General]
it says its a virus and i googled it but i didnt get results about it

**Comment by nesrak1** [comment]
https://github.com/nesrak1/UABEA/issues/354#issuecomment-2264153889

**Reply by yacmag** [reply]
what file do i download to get it?

**Reply by nesrak1** [reply]
The releases section (on the right side of the main repo page in green) in this repo is built from the code in this repo automatically. If you're referring to virustotal showing a few detections, false positives are always going to happen. Whether or not you want to trust it is up to you.
